### ISSUE TITLE
Displaying emoji in middle of markdown formatted text

### ISSUE THREAD

**Issue by Tunous** [divadretlaw/EmojiText#19]
I would like to format part of text that might contain emoji using markdown. 

Current implementation doesn't allow me to do that as anywhere where there is an emoji formatting breaks. Is there a known workaround? What I want to achieve is `<bold text with emoji> <regular text>` combined into correctly wrapping text.

---

Non working example:

```swift
func test_EmojiInMarkdown() async throws {
    let view = EmojiText(markdown: "**Hello :mastodon:** _Mastodon :mastodon:_ :mastodon:", emojis: [Emojis.mastodon])
    await assertSnapshot(matching: view, as: .rendered(size: CGSize(width: 300, height: 100), delay: 2))
}
```

![test_EmojiInMarkdown 1](https://github.com/divadretlaw/EmojiText/assets/5156340/6373df5f-3394-496a-b567-cb97c5e0fd6e)

**Comment by divadretlaw** [comment]
No, that is currently not possible but I have some ideas in mind but that will require some changes. I will see what I can do, hopefully it can be done without breaking changes.

**Comment by divadretlaw** [comment]
Proof of Concept, from the example test you provided:

![test_EmojiInMarkdown 1](https://github.com/divadretlaw/EmojiText/assets/6899256/f08a9eca-fa1c-4d52-9b99-8074336f96f4)

**Comment by divadretlaw** [comment]
Should be fixed with `3.3.0`. If there are any issues please let me know
